test/errors/multipleQuestionArgs-brief.good

```diff
@@ -0,0 +1,3 @@
+error in multipleQuestionArgs.chpl:4: cannot have ? more than once in a call
```

Comment:
I think `?` should be in quotes (`'?'`) as it has a particular non-English-grammar meaning in this context.